### ISSUE TITLE
`UI Position` property should be honored for the page `Tags` ?

### ISSUE THREAD

**Issue by cannibalox** [logseq/db-test#145]
### Search first

- [X] I searched and no similar issues were found

### What Happened?

currently, if we go to page `Tags` and click `configure Property` > `UI Position` to a non-default setting (different than block-property`, the tags are displayed twice : at the position set by the configuration option and at the end of the block :

![image](https://github.com/user-attachments/assets/7940195c-7c66-450e-ab1e-7ba4832636b3)

* see below using 'below the block`
![image](https://github.com/user-attachments/assets/d64810c5-8cdc-485a-bbf2-6e59d8ac0302)

* using `end of the block` :
![image](https://github.com/user-attachments/assets/c673dc29-13e3-4c33-8331-42fbb1f5e62d)

the behavior when clicking a #tag is different between the `tags: #mytag` and `#mytag`.  

### Reproduce the Bug

go to the page `Tags` and try different `UI Position` 

### Expected Behavior

* I would expect the default option to be `end of the block` rather than `block perperty` , with the current display position :
```
* my block                                                                #mytag1 #mytag2
```
* I would expect `UI Position > block property` to display the tags inside the block property:
```
* myblock
      tags: mytag1, mytag2
      prop: my prop
 ```
 * and same for the other options
 
 so basically, honor the user configuration, without showing the tags twice (at the endof the block)
 
 alternatively, maybe any `#Tag` should have the `UI Position` option that includes `inline` (which would complement the `ctrl enter`) ? 

### Browser, Desktop or Mobile Platform Information

logseq db 20241018 on win11

### Are you willing to submit a PR? If you know how to fix the bug.

- [ ] I'm willing to submit a PR (Thank you!)

**Comment by tiensonqin** [comment]
Closing as stale: no replies in over 3 months. Please reopen if this is still happening on a current 2.0 / nightly build.
